README.md

```diff
@@ -94,12 +94,23 @@ Install **both** of the following:
    - `calculation-example.js` contains JavaScript code that will be executed by each node of the DON. This example performs complex calculations but no API requests.
-   - `API-request-example.js` which fetches data from APIs before processing the data <br><br>
-6. Simulate the JavaScript code for the request locally to ensure it produces the expected output:<br>`npx hardhat functions-simulate-script`<br><br>
+   - `./API-request-example.js` which fetches data from APIs before processing the data <br><br>
+6. Locally simulate the execution of your JavaScript source by using:<br>`npx hardhat functions-simulate-script`.
+
 7. Deploy and verify the consumer contract to an actual blockchain network by running:<br>`npx hardhat functions-deploy-consumer --network network_name_here --verify true`<br>**Note**: Make sure `<explorer>_API_KEY` is set if using `--verify true`, depending on which network is used.<br><br>
-8. Create and fund a new Functions billing subscription. Make a note of your subscription Id as you will need it for most commands. Then authorize your deployed consumer contract to use that subscription by running:<br> `npx hardhat functions-sub-create --network network_name_here --amount LINK_funding_amount_here --contract 0xDeployed_consumer_contract_address_here`
-   <br>**Note**: Ensure your wallet has a sufficient LINK balance before running this command. Testnet LINK can be obtained at <a href="https://faucets.chain.link/">faucets.chain.link</a>.<br><br>
-9. Make an on-chain request by running:<br>`npx hardhat functions-request --network network_name_here --contract 0xDeployed_consumer_contract_address_here --subid subscription_id_number_here`
+8. Create and fund a new Functions billing subscription, and then authorize your deployed consumer contract to use that subscription by running:<br> `npx hardhat functions-sub-create --network network_name_here --amount LINK_funding_amount_here --contract 0xDeployed_client_contract_address_here`<br><br>**Note**: Ensure your wallet has a sufficient LINK balance before running this command. Testnet LINK can be obtained at <a href="https://faucets.chain.link/">faucets.chain.link</a>.
+   **Note**: also make a note of your subscription Id as you will need it for most commands. <br><br>
+9. Make an on-chain request by running:<br>`npx hardhat functions-request --network network_name_here --contract 0xDeployed_client_contract_address_here --subid subscription_id_number_here`. You will see a confirmation request, so hit `Y` and press enter. Once the request is fulfilled the console will show the response (decoded into the relevant return type) from the execution of your custom JS script.
+
+10. You can also query the response that was stored in your Functions Consumer contract by runnning `npx hardhat functions-read --contract consumer_contract_address --network  your_network_name`
```

Comment:
Double check that this works. I might have screwed something up locally, but I think I had an error here...

Reply:
works as at my current version which is one behind the lint fix pushed in 4e165605ba124c398d8b99d2d9a95e6d2fc39b8f